tool/planet/agent.go

```diff
@@ -364,6 +364,7 @@ func runAgent(conf *agent.Config, monitoringConf *monitoring.Config, leaderConf
 		}
 	}
 
+	go startSerfReconciler(ctx, &conf.SerfConfig)
```

Comment:
I would modify the algorithm so it runs only on master nodes, otherwise we will be hammering API server in large clusters. We need to start keeping scaling in mind now.